packages/swap/src/routes/quote.ts

```diff
@@ -63,6 +63,13 @@ const quote = (io: Server) => {
 
       logger.info('received a quote request', { query: req.query });
 
+      // detect if ingress and egress fees are exposed as gas asset amount or fee asset amount
+      // https://github.com/chainflip-io/chainflip-backend/pull/4497
+      // TODO: remove this once all networks are upraded to 1.3
+      const ingressEgressFeeIsGasAssetAmount =
+        (await getIngressFee({ chain: 'Ethereum', asset: 'FLIP' })) ===
```

Comment:
the `cf_environment` call inside of `getIngressFee` i cached, so this should not cause any additional requests